<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPve</class>
 <widget class="QWidget" name="FormPve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>五子棋——人机对战</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Pve.qrc">
    <normaloff>:/new/prefix1/Image/Icon/图标.jpg</normaloff>:/new/prefix1/Image/Icon/图标.jpg</iconset>
  </property>
  <widget class="QLabel" name="background_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1250</width>
     <height>900</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Image/Background/Gme1.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ChessBoard" name="checkerboard_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>840</width>
     <height>840</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Image/Background/check.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mainMenu_Button">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>500</y>
     <width>155</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Image/Button/150.50.jpg);</string>
   </property>
   <property name="text">
    <string>返回主菜单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_Button">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>600</y>
     <width>155</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Image/Button/150.50.jpg);</string>
   </property>
   <property name="text">
    <string>保存棋局</string>
   </property>
  </widget>
  <widget class="QPushButton" name="undo_Button">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>700</y>
     <width>155</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Image/Button/150.50.jpg);</string>
   </property>
   <property name="text">
    <string>悔棋</string>
   </property>
  </widget>
  <widget class="QPushButton" name="restart_Button">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>800</y>
     <width>155</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Image/Button/150.50.jpg);</string>
   </property>
   <property name="text">
    <string>重新开始</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>400</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <item>
    <property name="text">
     <string>简单人机</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>中等人机</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>困难人机</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>噩梦人机</string>
    </property>
   </item>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChessBoard</class>
   <extends>QLabel</extends>
   <header location="global">chessboard.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Pve.qrc"/>
 </resources>
 <connections/>
</ui>
